CMakeLists.txt

```diff
@@ -16,16 +16,12 @@ else()
     message(STATUS "Found GLFW ${GLFW_PREFIX}")
 endif()
 
-list(APPEND GLFW_LDFLAGS
-    "-framework OpenGL"
-    "-framework Cocoa"
-    "-framework IOKit"
-    "-framework CoreFoundation"
-    "-framework CoreVideo")
-
 include_directories(include)
 
 add_executable(${EXECUTABLE_NAME} tests/main.cpp include/isect2d.h)
 
-target_link_libraries(${EXECUTABLE_NAME} ${GLFW_LDFLAGS})
+target_link_libraries(${EXECUTABLE_NAME} ${GLFW_STATIC_LIBRARIES})
```

Comment:
Oh whoops, I think it worked on mine because I installed the 'static'
version of glfw. I'll correct that!

On Thu, Jun 25, 2015 at 3:39 PM, Karim Naaji <notifications@github.com>
wrote:

> In CMakeLists.txt
> <https://github.com/tangrams/isect2d/pull/3#discussion_r33295935>:
>
> >  include_directories(include)
> >
> >  add_executable(${EXECUTABLE_NAME} tests/main.cpp include/isect2d.h)
> >
> > -target_link_libraries(${EXECUTABLE_NAME} ${GLFW_LDFLAGS})
> > +target_link_libraries(${EXECUTABLE_NAME} ${GLFW_STATIC_LIBRARIES})
>
> I'm having missing links on glfw3 using static libraries on osx, does this
> happen to you?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/isect2d/pull/3/files#r33295935>.
>